packages/gestalt/src/Masonry.js

```diff
@@ -374,6 +381,7 @@ export default class Masonry<T> extends React.Component<Props<T>, State<T>> {
    */
   reflow() {
     this.props.measurementStore.reset();
```

Comment:
need to check if `this.props.measurementStore` is defined first

Reply:
So smart! Fixed.